clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/StoreRepushCommand.java

```diff
@@ -0,0 +1,276 @@
+package com.linkedin.venice.datarecovery;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.commons.lang.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+/**
+ * StoreRepushCommand contains the details of executing/processing repush command.
+ * We expect the command to comply with the following contract:
+ *
+ * Input:
+ *    venice-tools repush kafka --store store_name --fabric source_fabric --force --password user_credentials
+ * Output:
+ *    success: link_to_running_task
+ *    failure: failure_reason
+ */
+
+public class StoreRepushCommand {
+  private static final Logger LOGGER = LogManager.getLogger(StoreRepushCommand.class);
+
+  // Store name.
+  private String store;
+  private Params params;
+  private Result result;
+  private List<String> shellCmd;
+
+  public StoreRepushCommand() {
+  }
+
+  public StoreRepushCommand(String store, Params params) {
+    this.store = store;
+    this.params = params;
+    this.shellCmd = generateShellCmd();
+  }
+
+  public void setParams(Params params) {
+    this.params = params;
+  }
+
+  public String getStore() {
+    return store;
+  }
+
+  public Result getResult() {
+    return result;
+  }
+
+  private List<String> generateRepushCommand() {
+    List<String> cmd = new ArrayList<>();
+    cmd.add(this.params.command);
+    cmd.add(this.params.extraCommandArgs);
+
+    if (store != null) {
+      cmd.add("--store " + store);
+    }
+
```

Comment:
Also, should this arg be `--fabric` or `--source-fabric`? I'm okay with either way

Reply:
I plan to use the admin-tool's EXTRA_COMMAND_ARGS to capture `--fabric ` or `--source-fabric` parts and pass it to `venice-tools` directly, so we don't need a special handle for them. But,  for `--store`, it is a bit different, because I would allow the admin-tool's `--store` to take multiple store names, separated by a predefined separator as input. When calling the underlining `venice-tools`, today it should only take one store as input, and that's what it does here.